.kodiak.toml

```diff
@@ -0,0 +1,84 @@
+# This file may be out-of-date and thus may not contain all of the possible configuration settings. See the [Kodiak documentation](https://github.com/chdsbd/kodiak/blob/master/README.md) and the [config schema](https://github.com/chdsbd/kodiak/blob/master/kodiak/test/fixtures/config/config-schema.json) for more info.
+# .kodiak.toml
+# version is the only required setting in a kodiak config.
+# it must be set to 1
+version = 1
+
+[merge]
+# label to use to enable Kodiak to merge a PR
+automerge_label = ":white_check_mark: needs merge" # default: "automerge"
```

Comment:
Nit: Since you're overriding `automerge`, would also need to change `needs merge` to `:white_check_mark: needs merge` label in order for it listen

Reply:
Silly mistake from my end, got your point, thanks!